### PR TITLE
fix units of spell_length_distribution

### LINKED ISSUE #1003: spell_length_distribution has the wrong units
`xc.sdba.properties.spell_length_distribution` has units of the input, but it should have units of days.

### PR DESCRIPTION
**Pull request by juliettelavoie**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #1003 
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.
- [ ] `bumpversion patch` has been called on this branch
- [ ] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* `xc.sdba.properties.spell_length_distribution` has units of the input, but it should have units of days. This is fixed now.

### CODE REVIEW

File: HISTORY.rst
Comment (juliettelavoie) [reply HISTORY.rst:54]: oups!

**Comment by Zeitsperre** [comment]
@juliettelavoie @aulemahal I can merge this once approved, but for the checks to pass, we need to merge #1000 and #1002.

File: xclim/sdba/properties.py
Comment (juliettelavoie) [reply xclim/sdba/properties.py:312]: ok I will change it!

**Review by huard — APPROVED** [review]

**Pull request merged**
